### ISSUE TITLE
Make mod/OC tooltips immediate and permanent upon mouseover

### ISSUE THREAD

**Issue by MarHelmer** [phg49389/drg-weapons-calculator#17]
**Is your feature request related to a problem? Please describe.**
When mousing over a mod or OC, tooltips take about one second to display and then hide after about 3.5 seconds.  The display lag is frustrating when trying to compare options, and the display time is typically not long enough to read a mod's description.

**Describe the solution you'd like**
Change, or give the option to change, the tooltips to display immediately and permanently upon mouseover.  They will hide when the mouse leaves the mod's border.

**Comment by phg49389** [comment]
I tried making them immediate, and I didn't like it when moving the cursor around the GUI. It made the tooltips blip into and out of existence very quickly as I moved over the buttons. I set the delay down to 0.2 seconds so that moderate movement of the cursor won't be in the buttons long enough to trigger the ToolTips, but it will appear a lot faster. Additionally, I increased how long the ToolTips get displayed before hiding to 10 seconds. I don't think making the ToolTips permanent is the right solution, and this can always be increased to a delay greater than 10 seconds if that's still not enough.

Change implemented and will be in the next release; closing issue.
